ca_gazebo/models/create_camera/model.sdf

```diff
@@ -15,12 +15,12 @@
             <path>/tmp/camera_save_tutorial</path>
           <horizontal_fov>1.047</horizontal_fov>
           <image>
-            <width>5760</width>
-            <height>3240</height>
+            <width>1920</width>
```

Comment:
1920x1080 is quite a high resolution, would this work correctly on a lower resolution?